core/src/test/resources/application.conf

```diff
@@ -11,3 +11,5 @@ database = {
 messaging {
   mode = "test"
 }
+
+packages.versionFormat = ".+"
```

Comment:
Yeah we should fix that. The problem is this line https://github.com/advancedtelematic/rvi_sota_server/blob/819372d1855647d638c653bfdb3fbe7ee02c9645/common-data/src/main/scala/org/genivi/sota/data/PackageId.scala#L51-L51 We should use `.+` in case that config is not set.